src/components/Main/Footer.tsx

```diff
@@ -4,7 +4,7 @@ import strings from '../../utils/constants'
 const logo = require('../../../images/logos/logo_512.png').default
 
 const Footer: React.FC = () => (
-  <Container id="footer">
+  <Container id="footer" fluid>
```

Comment:
Not sure why, Footer on mobile look strange when looking at the Applications list:
![Screen Shot 2021-08-03 at 3 48 47 PM](https://user-images.githubusercontent.com/16461988/127954791-4a086d9b-b7a4-4269-8d07-971843290dfc.png)

![Screen Shot 2021-08-03 at 3 48 53 PM](https://user-images.githubusercontent.com/16461988/127954785-176a4d3c-c78d-40ee-87ca-05d0a19fbe6c.png)


In Dashboard look good:
![Screen Shot 2021-08-03 at 3 48 08 PM](https://user-images.githubusercontent.com/16461988/127954805-30fa5d37-4d34-45fe-b419-6bf155496dc5.png)

Reply:
I can't reproduce this. Looks okay for me. (in Firefox)

- I just tried it in Chrome, and it looks weird as, scaling is all wrong. Seems correct in Safari too